dragonchain/lib/broadcast.py

```diff
@@ -60,6 +63,22 @@ def send_receipt(block: "model.BlockModel") -> None:
         _log.error(f"[BROADCAST] ERROR: Couldn't broadcast receipt down to {dcid}! Ignoring")
 
 
+async def process_claims_backlog() -> None:
+    claims = await redis.hgetall_async("mq:failed-claims", decode=False)
+    claim_list = []
+    for _, value in claims.items():
+        claim_list.append(value)
+    _log.info(f"Claim backlog count: {len(claim_list)}")
+    if claim_list:
+        for claim_check_id in claim_list:
+            try:
+                matchmaking.resolve_claim_check(claim_check_id)
+            except Exception:
+                _log.exception("Failure to finalize claim in matchmaking.  Skipping the rest of the retry queue.")
+                return  # short-circuit and exit early, matchmaking still unreachable
+            redis.hdel_async("mq:failed-claims", claim_check_id)  # claim successfully finalized, remove from backlog
```

Comment:
why are you calling an async function without await? This will not work

Reply:
Didn't mean to be mixing these two, auto-complete fail.